addons/mail/static/src/models/thread_view/thread_view.js

```diff
@@ -67,6 +67,25 @@ function factory(dependencies) {
         // Private
         //----------------------------------------------------------------------
 
+        /**
+         * @private
+         * @returns {object}
+         */
+        _computeTextInputSendShortcuts() {
+            if (!this.thread) {
+                return;
+            }
+            if (
+                this.thread.model === 'mail.channel' &&
+                this.thread.mass_mailing ||
+                this.env.device.isMobile &&
+                this.thread.chatWindow
+            ) {
+                return ['ctrl-enter', 'meta-enter'];
+            }
```

Comment:
Try to reorganize the method to make sure there is no `or` condition hidden in the middle of `and` conditions because it is confusing and hard to notice.